cmd/migrate/migrate.go

```diff
@@ -0,0 +1,100 @@
+package migrate
+
+import (
+	"fmt"
+	"github.com/dymensionxyz/roller/cmd/utils"
+	"github.com/dymensionxyz/roller/config"
+	"github.com/dymensionxyz/roller/sequencer"
+	"github.com/dymensionxyz/roller/version"
+	"github.com/pelletier/go-toml"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+type VersionMigrator interface {
+	PerformMigration(rlpCfg config.RollappConfig) error
+}
+
+func NewVersionMigrator(version string) VersionMigrator {
+	switch version {
+	case "v0.1.4":
+		return &VersionMigratorV014{}
+	default:
+		return nil
+	}
+}
+
+type VersionData struct {
+	Major int
+	Minor int
+	Patch int
+}
+
+type VersionMigratorV014 struct{}
+
+func (v *VersionMigratorV014) PerformMigration(rlpCfg config.RollappConfig) error {
+	dymintTomlPath := sequencer.GetDymintFilePath(rlpCfg.Home)
+	dymintCfg, err := toml.LoadFile(dymintTomlPath)
+	if err != nil {
+		return err
+	}
+	sequencer.EnableDymintMetrics(dymintCfg)
+	return config.WriteTomlToFile(dymintTomlPath, dymintCfg)
+}
+
+func Cmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "migrate",
+		Short: "Migrates the roller configuration to the newly installed version.",
+		Run: func(cmd *cobra.Command, args []string) {
+			home := cmd.Flag(utils.FlagNames.Home).Value.String()
+			rlpCfg, err := config.LoadConfigFromTOML(home)
+			if err != nil {
+				utils.PrettifyErrorIfExists(err)
+				return
+			}
+			prevVersionData, err := GetVersionData(rlpCfg)
+			if err != nil {
+				utils.PrettifyErrorIfExists(err)
+				return
+			}
+			if prevVersionData.Major < 1 {
+				if prevVersionData.Minor < 2 {
+					if prevVersionData.Patch < 4 {
+						migrator := NewVersionMigrator("v0.1.4")
+						if migrator != nil {
```

Comment:
if migrator == nil
return error (e.g no migrations for this version from that version)